VictoriaAlbert/VictoriaAlbert/MuseumObjectTableViewCell.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  MusuemObjectTableViewCell.swift
+//  VictoriaAlbert
+//
+//  Created by Erica Y Stevens on 11/10/16.
+//  Copyright © 2016 Erica Stevens. All rights reserved.
+//
+
+import UIKit
+
+class MuseumObjectTableViewCell: UITableViewCell {
+    
+    // MARK: Outlets
+    
+    @IBOutlet weak var museumObjectImageView: UIImageView!
+    @IBOutlet weak var museumObjectTitleLabel: UILabel!
+    @IBOutlet weak var museumObjectSubtitleLabel: UILabel!
+    
+
+    // MARK: Built-In Methods
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+        
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+        // Configure the view for the selected state
+    }
+
+}
+
+    // MARK: Extensions
+
+//Note: I understand the concept of MVC (finally!), but when I tried to put this extension into its own file, I kept getting errors
```

Comment:
I'm guessing that you needed to import UIKit in the separate file which was only importing Foundation.